docs/spec/v2/README.md

```diff
@@ -0,0 +1,16 @@
+# helm.toolkit.fluxcd.io/v2
```

Comment:
`/doc/spec/README.md` design section should point here and not to `v2beta2` anymore.